server/test/e2e/asset.e2e-spec.ts

```diff
@@ -378,6 +385,58 @@ describe(`${AssetController.name} (e2e)`, () => {
     });
   });
 
+  describe('GET /asset/random', () => {
+    it('should require authentication', async () => {
+      const { status, body } = await request(server).get('/asset/random');
+
+      expect(status).toBe(401);
+      expect(body).toEqual(errorStub.unauthorized);
+    });
+
+    it('should return 1 random assets', async () => {
+      const { status, body } = await request(server)
+        .get('/asset/random')
+        .set('Authorization', `Bearer ${user1.accessToken}`);
+
+      expect(status).toBe(200);
+
+      const assets: AssetResponseDto[] = body;
+      expect(assets.length).toBe(1);
+      expect(assets[0].ownerId).toBe(user1.userId);
+      // assets owned by user1
+      expect(assets[0].id == asset1.id || assets[0].id == asset2.id || assets[0].id == asset3.id).toBeTruthy();
```

Comment:
I personally would prefer `expect([asset1.id, asset2.id, asset3.id]).toContain(assets[0].id)`

Reply:
I don't know why I didn't think of that, thank you :-)

Reply:
Well, it is somewhat counterintuitive :D